pkg/tables/tables.go

```diff
@@ -23,6 +23,7 @@ var (
 		{"App-Name", "Status.AppName"},
 		{"Bound-Volume", "Status.VolumeName"},
 		{"Capacity", "Spec.Capacity"},
+		{"Storage-Class", "Spec.Class"},
```

Comment:
Let's make this Volume-Class since the nomenclature is changing soon.

Reply:
Updated